index.html

```diff
@@ -8,62 +8,198 @@
+				</ul>
 			</div>
-	</section>
-	 <section class="works"></section>
-	 <section class="about"></section>
-	 <section class="contact"></section>
-  </main>
-  <footer></footer>
+			<button class="multiposts-btn">See Project</button>
+	  </div>
+      <div class="projects">
+        <div class="project-card">
+		  <h2 class="pjt-card-header heading">Profesional Art Printing Data</h2>
+          <p class="pjt-card-sub-header sub-heading">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+		  </p>
+		  <div >
+			<ul class="stack-list">
+		      <li><a href="#">html</a></li>
+			  <li><a href="#">bootstrap</a></li>
+			  <li><a href="#">Ruby</a></li>
+			</ul>
+		  </div>
+		  <button class="pjt-button">See Project</button>
+		</div>
+        <div class="project-card">
+		  <h2 class="pjt-card-header heading">Profesional Art Printing Data</h2>
+          <p class="pjt-card-sub-header sub-heading">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+		  </p>
+		  <div >
+			<ul class="stack-list">
+		      <li><a href="#">html</a></li>
+			  <li><a href="#">bootstrap</a></li>
+			  <li><a href="#">Ruby</a></li>
+			</ul>
+		  </div>
+		  <button class="pjt-button">See Project</button>
+		</div>
+        <div class="project-card">
+		  <h2 class="pjt-card-header heading">Profesional Art Printing Data</h2>
+          <p class="pjt-card-sub-header sub-heading">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+		  </p>
+		  <div >
+			<ul class="stack-list">
+		      <li><a href="#">html</a></li>
+			  <li><a href="#">bootstrap</a></li>
+			  <li><a href="#">Ruby</a></li>
+			</ul>
+		  </div>
+		  <button class="pjt-button">See Project</button>
+		</div>
+        <div class="project-card">
+		  <h2 class="pjt-card-header heading">Profesional Art Printing Data</h2>
+          <p class="pjt-card-sub-header sub-heading">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+		  </p>
+		  <div >
+			<ul class="stack-list">
+		      <li><a href="#">html</a></li>
+			  <li><a href="#">bootstrap</a></li>
+			  <li><a href="#">Ruby</a></li>
+			</ul>
+		  </div>
+		  <button class="pjt-button">See Project</button>
+		</div>
+        <div class="project-card">
+		  <h2 class="pjt-card-header heading">Profesional Art Printing Data</h2>
+          <p class="pjt-card-sub-header sub-heading">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+		  </p>
+		  <div >
+			<ul class="stack-list">
+		      <li><a href="#">html</a></li>
+			  <li><a href="#">bootstrap</a></li>
+			  <li><a href="#">Ruby</a></li>
+			</ul>
+		  </div>
+		  <button class="pjt-button">See Project</button>
+		</div>
```

Comment:
- [Optional] Kindly follow the Figma design.


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/205357093-fb569208-f1cf-401b-bec9-3c5090277be7.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/205357086-37afb7dd-4584-4d17-9dc1-dd43aba3625f.png" width="200">|


😎️